oddoor/models/oddoor_key_action.py

```diff
@@ -0,0 +1,19 @@
+# Copyright 2019 Creu Blanca
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class OddoorKeyAction(models.Model):
```

Comment:
This model should also have a datetime field to know when did the action happen

Reply:
create_date whould be it, isn't it?